src/renderer/src/components/Map.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Map as ReactMaplibre } from '@vis.gl/react-maplibre'
+import maplibregl from 'maplibre-gl'
+
+import 'maplibre-gl/dist/maplibre-gl.css'
+
+export function Map() {
+	const center = [-72.312023, -10.38787]
+
+	return (
+		<ReactMaplibre
+			mapLib={maplibregl}
+			initialViewState={{
+				longitude: center[0],
+				latitude: center[1],
+				zoom: 6,
+			}}
+			dragPan={true}
+			scrollZoom={true}
+			doubleClickZoom={true}
+			style={{ width: '100%', height: '100%' }}
+			mapStyle="https://demotiles.maplibre.org/style.json"
```

Comment:
although I initially suggested using the hook here, it turns out that MapLibre dropped support for `mapbox://` urls, which our default online style currently contains. there's a way of fixing that by using https://github.com/rowanwins/maplibregl-mapbox-request-transformer but it requires the renderer needing access to the the access token, which isn't ideal and probably prone to breaking under other circumstances. Will file an issue to fix the mapbox-specific urls in core but for now, i think it's okay to use the MapLibre demotiles url for the `mapStyle`, like you were doing before.

added an issue in core for this: https://github.com/digidem/comapeo-core/issues/982